pom.xml

```diff
@@ -136,7 +136,6 @@
         <version.org.bouncycastle>1.52</version.org.bouncycastle>
         <version.org.codehaus.jackson>1.9.13</version.org.codehaus.jackson>
         <version.org.codehaus.jettison>1.3.3</version.org.codehaus.jettison>
-        <version.org.codehaus.plexus.plexus-utils>3.0.21</version.org.codehaus.plexus.plexus-utils>
```

Comment:
was only needed as temporary workaround for making our build (transformers MavenUtil) work with maven 3.0.x and 3.1 at the same time. as they moved to eclipse aether.